vcpkg/contributing/documentation-style-guide.md

```diff
@@ -0,0 +1,189 @@
+---
+title: vcpkg Documentation Style Guide
+description: Writing style guidelines for vcpkg documentation contributors and AI assistants.
+author: vicroms
+ms.author: viromer
+ms.date: 08/19/2025
+ms.topic: concept-article
+---
+
+# Documentation Style Guide
+
+This guide establishes consistent writing standards for vcpkg documentation. These guidelines ensure
+clarity, accessibility, and maintainability across all documentation, whether written by human
+contributors or AI assistants.
+
+## Writing Principles
+
+### Voice and Tone
+
+- **Authoritative but approachable**: Be confident and knowledgeable while remaining accessible
```

Comment:
In my experience these things are already fairly confident and asking them to be even more confident may increase the risk of hallucinations but :shrug: